internal/eval/eval.go

```diff
@@ -3621,6 +3621,81 @@ func evalJSONNUMINCRBY(args []string, store *dstore.Store) []byte {
 	return clientio.Encode(resultString, false)
 }
 
+// evalJSONOBJKEYS retrieves the keys of a JSON object stored at path specified.
+// It takes two arguments: the key where the JSON document is stored, and an optional JSON path. 
+// It returns a list of keys from the object at the specified path or an error if the path is invalid.
+func evalJSONOBJKEYS(args []string, store *dstore.Store) []byte {
+	if len(args) < 1 {
+		return diceerrors.NewErrArity("JSON.OBJKEYS")
+	}
+
+	key := args[0]
+	// Default path is root if not specified
+	path := defaultRootPath
+	if len(args) > 1 {
+		path = args[1]
+	}
+
+	// Retrieve the object from the database
+	obj := store.Get(key)
+	if obj == nil {
+		return diceerrors.NewErrWithMessage("could not perform this operation on a key that doesn't exist")
```

Comment:
Thought was to make JSON.OBJKEYS consistent with Redis. Redis throws this error when we try to query with an unavailable key.
<img width="528" alt="image" src="https://github.com/user-attachments/assets/f7a907b7-3334-4261-8e91-e0fd1f1070b1"> (Redis)

Also, if we don't pass rootpath , i.e. '$', we get (nil) as response in Redis. In DiceDB, I observed in all commands, we consider rootPath ($) as defaultPath. So, even if we don't pass any path, we respond with same key DNE error. 

<img width="575" alt="image" src="https://github.com/user-attachments/assets/241cf459-f50b-47a0-8af9-9f5219a09a7d"> (DiceDB)


I am not sure why absence and presence of path is leading to different response in case of missing keys in case of Redis. 
<img width="546" alt="image" src="https://github.com/user-attachments/assets/6d439737-fe3e-4c75-a36f-be44c7e5b15e"> (Redis)

Please suggest if we should be responding (nil) in case of no path, and unavailable key.